packages/providers/src/core/PoapTokenApi/PoapTokenApi.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  PostClaimCodeResponse,
+  ClaimStatusResponse,
+  GetClaimCodeResponse,
+} from './../../ports/TokensApiProvider/Types/response';
+import { ClaimCodeInput } from './../../ports/TokensApiProvider/Types/input';
+import { AuthenticationProvider } from './../../ports/AuthenticationProvider/AuthenticationProvider';
+import { TokensApiProvider } from './../../ports/TokensApiProvider/TokensApiProvider';
+import axios from 'axios';
+
+const instance = axios.create({
+  timeout: 10000, // 5 seconds
+});
+
+const DROP_BASE_URL = 'https://api.poap.tech';
```

Comment:
Isn't this some soft of "default"?

```suggestion
const DEFAULT_DROP_BASE_URL = 'https://api.poap.tech';
```

```suggestion
const DROP_BASE_URL_DEFAULT = 'https://api.poap.tech';
```